### PR TITLE
New article

### PR DESCRIPTION
**Pull request by Favour-dgreat**
## Category

- [x] New article

## Contents of the Pull Request

> New article on Microsoft Power Pages

### CODE REVIEW

**Review by hugoabernier — CHANGES_REQUESTED** [review]
These changes should be reverted. Please make sure that you only affect the files related to your post.

File: content/post/understanding-data-connectors-with-microsoft-power-apps/index.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+Standard Connectors are connectors provided by Power Apps for many commonly used data sources. If Power Apps has a standard connector for the type of data source that you want to use, you should use that connector, those are what are referred to as Standard Custom Connectors.
+
+The image above showing popular connectors are all Standard Connectors. 
+
+But in a situation where you want to connect to other types of data sources, such as the one you built, then they are referred to as Custom Connectors 
+
+Check out this article for more on  Custom Connectors - [Register and use custom connectors](https://docs.microsoft.com/en-us/powerapps/maker/canvas-apps/register-custom-api)
+
+
+# Connecting to Microsoft Dataverse
```
Comment (hugoabernier) [inline content/post/understanding-data-connectors-with-microsoft-power-apps/index.md]: You should only have one level 1 heading in the post, and it is reserved for the page title. Please stick to level 2 (`##`) or above.

File: content/post/understanding-data-connectors-with-microsoft-power-apps/index.md [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+
+In very clear terms, "Data Connectors" connects or communicates with the data source in every power app. Data is stored in a data source, and you bring that data into your app by creating a connection.
+
+In Microsoft Power Apps a connector may provide tables of data or actions. Some connectors provide only tables, some provide only actions, and some provide both. Also, your connector may be either a **standard or custom connector**.
+
+
+### Popular Connectors used in Microsoft Power Apps
+
+
+![banner.webp](images/banner.webp)
```
Comment (hugoabernier) [inline content/post/understanding-data-connectors-with-microsoft-power-apps/index.md:30]: The alt tag should describe the image

File: content/post/understanding-data-connectors-with-microsoft-power-apps/index.md [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+tags: []
+# don't change
+type: "regular"
+---
+
+## Understanding Data Connectors with Microsoft Power Apps
+
+In today's article, I will be looking at working with Data Connectors while building with Microsoft Power Apps. 
+
+# What are Data Connectors?
```
Comment (hugoabernier) [inline content/post/understanding-data-connectors-with-microsoft-power-apps/index.md]: Level one headings are not allowed

File: content/post/understanding-data-connectors-with-microsoft-power-apps/index.md [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+
+Microsoft Dataverse lets you securely store and manage data that are used by business applications. 
+Data within Dataverse is stored within a set of tables. A table is a set of rows (formerly referred to as records) and columns (formerly referred to as fields/attributes). Each column in the table is designed to store a certain type of data, for example, name, age, salary, and so on.
+
+Now with the understanding of the Dataverse let us get into this codelab!
+
+## Building a simple record-keeping app with Google Sheets (a Standard Connector):
+1. [Sign in to Power Apps](https://powerapps.microsoft.com/en-gb/) If you don't already have a Power Apps account, click the Start Free button. 
+
+![image1.png](images/image1.png)
```
Comment (hugoabernier) [inline content/post/understanding-data-connectors-with-microsoft-power-apps/index.md:53]: Descriptive alt tag required

File: content/post/understanding-data-connectors-with-microsoft-power-apps/index.md [hunk 5]
```diff
@@ -0,0 +1,97 @@
...
+
+2. After successful sign in to Power Apps this is what you should have now! 
+
+![image2.png](images/image2.png)
```
Comment (hugoabernier) [inline content/post/understanding-data-connectors-with-microsoft-power-apps/index.md:57]: Descriptive alt tag required

File: content/post/understanding-data-connectors-with-microsoft-power-apps/index.md [hunk 6]
```diff
@@ -0,0 +1,97 @@
...
+![image1.png](images/image1.png)
+
+2. After successful sign in to Power Apps this is what you should have now! 
+
+![image2.png](images/image2.png) 
+Click on the Dataverse card. 
+
+3. You should have this screen now, showing previous connections. Then you click on New Connection
+
+![image3.png](images/image3.png)
```
Comment (hugoabernier) [inline content/post/understanding-data-connectors-with-microsoft-power-apps/index.md:62]: Descriptive alt tag required

File: content/post/understanding-data-connectors-with-microsoft-power-apps/index.md [hunk 7]
```diff
@@ -0,0 +1,97 @@
...
+
+4. If you followed the previous step properly, you should have this now, then click on Google Sheets and authenticate with your Google account. 
+
+![image4.png](images/image4.png)
```
Comment (hugoabernier) [inline content/post/understanding-data-connectors-with-microsoft-power-apps/index.md:66]: Descriptive alt tag required

File: content/post/understanding-data-connectors-with-microsoft-power-apps/index.md [hunk 8]
```diff
@@ -0,0 +1,97 @@
...
+Now you choose a dataset, from a set of previous google sheet documents or perhaps you search for the one you have in mind just as I have done here. 
+
+5. The next option is to choose a table or create a new one. 
+
+![image5.png](images/image5.png)
```
Comment (hugoabernier) [inline content/post/understanding-data-connectors-with-microsoft-power-apps/index.md:71]: Descriptive alt tag required

File: content/post/understanding-data-connectors-with-microsoft-power-apps/index.md [hunk 9]
```diff
@@ -0,0 +1,97 @@
...
+
+6. And finally you should have a very simple app for you with the data from the table you chose or created. Just as I have here too. 
+
+![image6.png](images/image6.png)
```
Comment (hugoabernier) [inline content/post/understanding-data-connectors-with-microsoft-power-apps/index.md:75]: Descriptive alt tag required

File: content/post/understanding-data-connectors-with-microsoft-power-apps/index.md [hunk 10]
```diff
@@ -0,0 +1,97 @@
...
+
+That is the end of this codelab and hope you were able to follow it properly. 
+
+I will also be ending this article here now, leaving you with resources for further study. 
+
+# Resources
+1. [Overview of connectors for canvas apps](https://docs.microsoft.com/en-us/powerapps/maker/canvas-apps/connections-list)
```
Comment (hugoabernier) [inline content/post/understanding-data-connectors-with-microsoft-power-apps/index.md:82]: Please remove `/en-us/` from URLs so the links are relevant for non-US visitors

File: content/post/understanding-data-connectors-with-microsoft-power-apps/index.md [hunk 11]
```diff
@@ -0,0 +1,97 @@
...
+2. [Use custom connectors in a Power Apps canvas app](https://docs.microsoft.com/en-us/learn/modules/use-custom-connectors-in-powerapps-canvas-app/)
```
Comment (hugoabernier) [inline content/post/understanding-data-connectors-with-microsoft-power-apps/index.md:83]: No `/en-us/` please

File: content/post/understanding-data-connectors-with-microsoft-power-apps/index.md [hunk 12]
```diff
@@ -0,0 +1,97 @@
...
+### Popular Connectors used in Microsoft Power Apps
+
+
+![banner.webp](images/banner.webp)
+
+
+### Standard and Custom Connectors
+Standard Connectors are connectors provided by Power Apps for many commonly used data sources. If Power Apps has a standard connector for the type of data source that you want to use, you should use that connector, those are what are referred to as Standard Custom Connectors.
+
+The image above showing popular connectors are all Standard Connectors.
```
Comment (hugoabernier) [inline content/post/understanding-data-connectors-with-microsoft-power-apps/index.md:36]: Please avoid terms like `above` for accessibility reasons.

File: content/post/understanding-data-connectors-with-microsoft-power-apps/index.md [hunk 13]
```diff
@@ -0,0 +1,97 @@
...
+
+
+### Standard and Custom Connectors
+Standard Connectors are connectors provided by Power Apps for many commonly used data sources. If Power Apps has a standard connector for the type of data source that you want to use, you should use that connector, those are what are referred to as Standard Custom Connectors.
+
+The image above showing popular connectors are all Standard Connectors. 
+
+But in a situation where you want to connect to other types of data sources, such as the one you built, then they are referred to as Custom Connectors 
+
+Check out this article for more on  Custom Connectors - [Register and use custom connectors](https://docs.microsoft.com/en-us/powerapps/maker/canvas-apps/register-custom-api)
```
Comment (hugoabernier) [inline content/post/understanding-data-connectors-with-microsoft-power-apps/index.md:40]: No `en-us` in URLs please

**Review by hugoabernier — CHANGES_REQUESTED** [review]
Great article, only need to fix a few accessibility issues and mardown issues. Also, please revert changes that are not related to your post (see the other commits in this pull request)

**Comment by Favour-dgreat** [comment]
I have made necessary adjustments.

**Comment by hugoabernier** [comment]
> Oh thank You so much. Is it live on the community blog now?
 Yes, sorry, I should have been clearer! Thank you for your article!

**Comment by Favour-dgreat** [comment]
I could only find the previously published article. The other article I
sent in have not been published yet?

**Comment by Favour-dgreat** [comment]
The second article the one that talked on Power Pages

**Comment by hugoabernier** [comment]
Can you give me the article name or slug?

**Comment by Favour-dgreat** [comment]
Quick Overview of Microsoft Power Pages - that’s the title

**Comment by hugoabernier** [comment]
Ok, I merged the article. Please try to keep your pull requests to a single article in your future pull requests so we can process them without further issues.

**Pull request closed without merging**